soc/xtensa/intel_adsp/ace_v1x/ace-ipc-regs.h

```diff
@@ -15,6 +15,15 @@
  * connection.
  */
 
+/**
+ * @brief Set TDA busy bit done.
+ *
+ * On ACE SoC family boards TDA bit 31 (BUSY) during IPC doorbell acknowledgment
+ * must be cleared (!), not set (in contrary to CAVS SoC family boards).
+ * This clears BUSY on the other side of the connection in IDR register.
+ */
+#define ACE_IPC_TDA1X_DONE BIT(0)
```

Comment:
There is no such thing as bit DONE at position 0 in TDA registry according to ACE documentation. TDA has only BUSY bit at position 31 and rest is MSG field (bits 30..0). Actually the comment above is correct. So this define is just wrong and misleading.